app/javascript/dashboard/routes/dashboard/helpcenter/components/Header/EditArticleHeader.vue

```diff
@@ -161,4 +259,7 @@ export default {
     }
   }
 }
+::v-deep.button-group .button {
```

Comment:
We could remove these overrides?

Reply:
If we move this font size will be small. For small-size buttons, the font size is mini. In the button-group class the font size will be small.
<img width="397" alt="image" src="https://user-images.githubusercontent.com/64252451/187932947-f797f94e-a132-496c-a95e-1907bc5e4e29.png">

Reply:
@nithindavid

Reply:
Let it be like this, This should be fixed on the global button group classes

Reply:
Done https://github.com/chatwoot/chatwoot/pull/5365/commits/7492e0c52571b57c567e45aeface7cb38726699e